docs/release-notes/0.10/release-0.10.md

```diff
@@ -0,0 +1,49 @@
+# .NET for Apache Spark 0.10 Release Notes
+
+### New Features and Improvements
+
+* Expose internal API to get Spark version on the driver side ([#429](https://github.com/dotnet/spark/pull/429))
+* Add `SPARK_VERSION_OVERRIDE` environment variable ([#432](https://github.com/dotnet/spark/pull/432))
+
+### Breaking Changes
+
+* None, but `SPARK_VERSION_OVERRIDE` environment variable is added to override the Spark version.
```

Comment:
we can remove "but .." since this is the change only on the driver side, and doesn't break the old behavior.

Reply:
Yea, make sense. Removed.